onnxruntime/python/tools/transformers/models/llama/requirements-cuda.txt

```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
+--extra-index-url https://download.pytorch.org/whl/cu118
+torch>=2.0.1+cu118
```

Comment:
not specify cuda version for package (for example, next version of torch will support cuda 12.*).

Reply:
Changed to comment so the user installs the desired package manually